lib/src/rules/overridden_fields.dart

```diff
@@ -125,7 +128,7 @@ class _Visitor extends SimpleAstVisitor<void> {
 
     node.fields.variables.forEach((VariableDeclaration variable) {
       final field = _getOverriddenMember(variable.declaredElement);
-      if (field != null) {
+      if (field != null && (!nnbdEnabled || !field.isAbstract)) {
```

Comment:
I don't believe that the condition needs to be this complex. If null safety is not enabled then the field can't be abstract, so testing the abstractness of the field should be sufficient. Let's drop the feature check.

Reply:
Good catch.